src/AvtaleSide/GodkjenningSteg/Oppsummering/instruks/instruks.less

```diff
@@ -2,12 +2,20 @@
     &__subheader {
         padding: 0 0 0 1.25rem;
     }
-
+    &__instrukslist {
+        margin-top: 1rem;
+        padding-left: 1.25rem;
```

Comment:
Hva er forskjell på padding og margin her, siden du bruker begge?

Reply:
Blir ikke noe forskjell her, endret til marging begge to.